packages/async-flow/src/type-guards.js

```diff
@@ -65,4 +68,6 @@ export const LogEntryShape = M.or(
   ],
   // ['checkReturn', M.number(), M.any()],
   // ['checkThrow', M.number(), M.any()],
-);
+];
+
+export const LogEntryShape = M.or(...LogEntryAlternatives);
```

Comment:
What is gained from the extra indirection here of first forming the list of alternatives and then feeding them to an `M.or`? Do you expect to make some other use of `LogEntryAlternatives`?